deploy/rp-development.json

```diff
@@ -35,6 +35,41 @@
             "location": "global",
             "apiVersion": "2018-05-01"
         },
+        {
+            "properties": {
+                "tenantId": "[subscription().tenantId]",
+                "sku": {
+                    "family": "A",
+                    "name": "standard"
+                },
+                "accessPolicies": [
+                    {
+                        "tenantId": "[subscription().tenantId]",
+                        "objectId": "[parameters('fpServicePrincipalId')]",
+                        "permissions": {
+                            "secrets": [
+                                "get"
+                            ],
+                            "certificates": [
+                                "create",
+                                "delete"
+                            ]
+                        }
+                    },
+                    {
+                        "tenantId": "[subscription().tenantId]",
+                        "objectId": "[parameters('adminObjectId')]"
+                    }
+                ]
+            },
+            "name": "[concat(parameters('keyvaultPrefix'), '-clusters')]",
+            "type": "Microsoft.KeyVault/vaults",
+            "location": "[resourceGroup().location]",
+            "tags": {
+                "vault": "clusters"
+            },
+            "apiVersion": "2016-10-01"
```

Comment:
How so we use `v7.0` client SDK and this version?

Reply:
v7.0 is the latest API by which you talk to a key vault (get key, create certificate, etc.)
2016-10-01 is (I think) the latest ARM API by which you create or configure a key vault